gqlauth/migrations/0002_alter_userstatus_options.py

```diff
@@ -1,8 +1,17 @@
 # Generated by Django 4.1.7 on 2023-05-07 15:38
 
+from django.contrib.auth import get_user_model
```

Comment:
Why have you modified the migrations?

Reply:
If you don't apply this package from scratch and change it during server operation, you will get an error because the existing users don't have a userstatus value, so you need to fill in the userstatus value artificially. I think it would be more convenient for people to use this process in the migration script.